demo/api/src/app.module.ts

```diff
@@ -91,10 +91,10 @@ export class AppModule {
                 UserPermissionsModule.forRootAsync({
                     useFactory: (userService: UserService, accessControlService: AccessControlService) => ({
                         availableContentScopes: [
-                            { domain: "main", language: "de" },
-                            { domain: "main", language: "en" },
-                            { domain: "secondary", language: "en" },
-                            { domain: "secondary", language: "de" },
+                            { domain: { value: "main", label: "MAIN" }, language: { value: "en", label: "English" } },
```

Comment:
how would this look like for a content-scope structures like this:
`{ channelId: "UUID", organizationId: null }` and `{ channelId: null, organizationId: "UUID" }`

like this?

```
availableContentScopes: [
    { channelId: { value: channel.id, label: channel.name }, organizationId: { value: null, label: null } },
    { channelId: { value: null, label: null }, organizationId: { value: organization.id, label: organization.title } },
]
```